posts/inside-rust/2022-07-29-compiler-team-2022-midyear-report.md

```diff
@@ -0,0 +1,800 @@
+---
+layout: post
+title: Rust Compiler Midyear Report for 2022
+author: Felix Klock, Wesley Wiser
+description: "The compiler team's midyear report on its ambitions for 2022."
+team: The Compiler Team <https://www.rust-lang.org/governance/teams/compiler>
+---
+
+# Rust Compiler Midyear Report for 2022
+
+Back in February, the compiler team [posted a collection][ambitions-post] of
+concrete initiatives and hopeful aspirations for this year. This post is a
+midyear report summarizing the progress so far on all of those items.
+
+[ambitions-post]: /inside-rust/2022/02/22/compiler-team-ambitions-2022.html
+
+(The original plan was to have this report out by the end of June, but there
+were extenuating circumstances that delayed the post until now.)
```

Comment:
I would consider dropping this sentence. It doesn't really add to the content of the post and we're only about 1-2 weeks off from the halfway point of 2/22 - 12/31 anyway.

Reply:
```suggestion
```